internal/olm/operator/registry/operator_installer.go

```diff
@@ -119,19 +119,6 @@ func (o OperatorInstaller) UpgradeOperator(ctx context.Context) (*v1alpha1.Clust
 
 	log.Infof("Found existing subscription with name %s and namespace %s", subscription.Name, subscription.Namespace)
 
-	// todo: attempt #1 to trigger install plan for the subscription and
-	// to make it detect catalog changes, as per OLM suggestion
-	if err := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
```

Comment:
Is this not necessary anymore?

Reply:
No, it's not. This was an initial attempt for OLM to trigger the update based on OLM suggestion. But the issue was something else that got fixed with the previous implementation PR, so wanted to clean this up.